javatests/google/registry/export/ExportConstantsTest.java

```diff
@@ -52,6 +54,17 @@
       "=================================================================================",
       "");
 
+  private static final String UPDATE_REPORTING_KINDS_INSTRUCTIONS_TEMPLATE = Joiner.on('\n').join(
```

Comment:
I think it'd be a little cleaner to keep using the existing template and just add a third substitution parameter in the first line of text:
"Your changes affect the list of %s in the golden file:"

Then just pass "backed-up kinds" and "reporting kinds" in the two callsites.  Less overall duplication that way.